ethereumetl/streaming/eth_streamer_adapter.py

```diff
@@ -88,7 +95,23 @@ def export_all(self, start_block, end_block):
             if EntityType.CONTRACT in self.entity_types else []
         enriched_tokens = enrich_tokens(blocks, tokens) \
             if EntityType.TOKEN in self.entity_types else []
-
+        
+        # enrich trace 
+        if self.node_client == "geth":
+            # 1) firsh enrich to get transactions_hash
+            enriched_traces_by_blocks_transactions = enrich_traces_with_blocks_transactions(blocks, traces, transactions)
+            # 2) 补充trace_status
+            # dict -> trace object
+            trs = []
```

Comment:
`trs = [trace_mapper.dict_to_trace(tr) for tr in enriched_traces_by_blocks_transactions]`